src/GraphQL/IdAttribute.cs

```diff
@@ -6,7 +6,7 @@ namespace GraphQL
     /// <summary>
     /// Specifies that a property will be mapped to <see cref="IdGraphType"/>.
     /// </summary>
-    [AttributeUsage(AttributeTargets.Property)]
+    [AttributeUsage(AttributeTargets.Property | AttributeTargets.Method | AttributeTargets.Field, Inherited = false)]
```

Comment:
Why do you use `Inherited=false` here and in other places?

```csharp
public class Person
{
  [Id]
  public virtual string Name {get;set; } // <-- IdGraphType
}

public class Manager : Person
{
  public override Name {get;set} // <-- StringGraphType
}

Reply:
I misunderstood what the `Inherited` property was for.  Certainly we would want it set to `true`.  I'll fix it, and add tests.

Reply:
Done